Userland/Libraries/LibWeb/DOM/Document.cpp

```diff
@@ -1182,4 +1182,21 @@ bool Document::has_focus() const
     return true;
 }
 
+void Document::increment_number_of_things_delaying_the_load_event(Badge<DocumentLoadEventDelayer>)
```

Comment:
Is this called for loading resources? It seems like `DocumentLoadEventDelayer` is used for `<script>` and `<link>` tags that delay parsing (i.e. the page is "delayed" while a script is running), not for resources we are downloading

Reply:
The intent for now was just to capture the number of blocking resources and give an idea that something is happening. Fully tracking all resource loads was a little bit more involved